Implement the plugin interface for golangci-lint (#234)
uber-go/nilaway: README.md

nilaway -include-pkgs="<YOUR_PKG_PREFIX>,<YOUR_PKG_PREFIX_2>" ./...
```

### golangci-lint (>= v1.57.0)

> NilAway, in its current form, still reports a fair number of false positives. This makes NilAway
> fail to be merged with [golangci-lint][golangci-lint] and be offered as a linter

Reviewer: ```suggestion
NilAway, in its current form, can report false positives. This unfortunately hinders its immediate merging in [golangci-lint][golangci-lint] and be offered as a linter
```
